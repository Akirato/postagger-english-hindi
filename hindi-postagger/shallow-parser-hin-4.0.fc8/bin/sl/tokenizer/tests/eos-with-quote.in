'सुभाष' ने कहा - "तुम मुझे खून दो, मैं तुम्हें आजादी दूंगा।" रमेश कुमार कक्षा ११वीं का छात्र है. "सुभाष" ने कहा - 'तुम मुझे खून दो, मैं तुम्हें आजादी दूंगा।'
'सुभाष' ने कहा - "तुम मुझे खून दो, मैं तुम्हें आजादी दूंगा?" रमेश कुमार कक्षा ११वीं का छात्र है. "सुभाष" ने कहा - 'तुम मुझे खून दो, मैं तुम्हें आजादी दूंगा?'
'सुभाष' ने कहा - "तुम मुझे खून दो, मैं तुम्हें आजादी दूंगा." रमेश कुमार कक्षा ११वीं का छात्र है. "सुभाष" ने कहा - "तुम मुझे खून दो, मैं तुम्हें आजादी दूंगा." 
