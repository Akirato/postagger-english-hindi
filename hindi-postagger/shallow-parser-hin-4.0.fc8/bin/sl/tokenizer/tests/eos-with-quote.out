<Sentence id="1">
1	'	unk
2	सुभाष	unk
3	'	unk
4	ने	unk
5	कहा	unk
6	-	unk
7	"	unk
8	तुम	unk
9	मुझे	unk
10	खून	unk
11	दो	unk
12	,	unk
13	मैं	unk
14	तुम्हें	unk
15	आजादी	unk
16	दूंगा	unk
17	।	unk
18	"	unk
</Sentence>

<Sentence id="2">
1	रमेश	unk
2	कुमार	unk
3	कक्षा	unk
4	११वीं	unk
5	का	unk
6	छात्र	unk
7	है	unk
8	.	unk
</Sentence>

<Sentence id="3">
1	"	unk
2	सुभाष	unk
3	"	unk
4	ने	unk
5	कहा	unk
6	-	unk
7	'	unk
8	तुम	unk
9	मुझे	unk
10	खून	unk
11	दो	unk
12	,	unk
13	मैं	unk
14	तुम्हें	unk
15	आजादी	unk
16	दूंगा	unk
17	।	unk
18	'	unk
</Sentence>

<Sentence id="4">
1	'	unk
2	सुभाष	unk
3	'	unk
4	ने	unk
5	कहा	unk
6	-	unk
7	"	unk
8	तुम	unk
9	मुझे	unk
10	खून	unk
11	दो	unk
12	,	unk
13	मैं	unk
14	तुम्हें	unk
15	आजादी	unk
16	दूंगा	unk
17	?	unk
18	"	unk
</Sentence>

<Sentence id="5">
1	रमेश	unk
2	कुमार	unk
3	कक्षा	unk
4	११वीं	unk
5	का	unk
6	छात्र	unk
7	है	unk
8	.	unk
</Sentence>

<Sentence id="6">
1	"	unk
2	सुभाष	unk
3	"	unk
4	ने	unk
5	कहा	unk
6	-	unk
7	'	unk
8	तुम	unk
9	मुझे	unk
10	खून	unk
11	दो	unk
12	,	unk
13	मैं	unk
14	तुम्हें	unk
15	आजादी	unk
16	दूंगा	unk
17	?	unk
18	'	unk
</Sentence>

<Sentence id="7">
1	'	unk
2	सुभाष	unk
3	'	unk
4	ने	unk
5	कहा	unk
6	-	unk
7	"	unk
8	तुम	unk
9	मुझे	unk
10	खून	unk
11	दो	unk
12	,	unk
13	मैं	unk
14	तुम्हें	unk
15	आजादी	unk
16	दूंगा	unk
17	.	unk
</Sentence>

<Sentence id="8">
1	"	unk
2	रमेश	unk
3	कुमार	unk
4	कक्षा	unk
5	११वीं	unk
6	का	unk
7	छात्र	unk
8	है	unk
9	.	unk
</Sentence>

<Sentence id="9">
1	"	unk
2	सुभाष	unk
3	"	unk
4	ने	unk
5	कहा	unk
6	-	unk
7	"	unk
8	तुम	unk
9	मुझे	unk
10	खून	unk
11	दो	unk
12	,	unk
13	मैं	unk
14	तुम्हें	unk
15	आजादी	unk
16	दूंगा	unk
17	.	unk
</Sentence>

<Sentence id="10">
1	"	unk
</Sentence>

